### DISCUSSION TITLE
Hands not visible

### DISCUSSION THREAD

**Discussion by Maddogofgithub** [Okabintaro/SubmersedVR#54] [Q&A] [answered]

**Comment by ishanwadhwani** [comment]
Can you elaborate your issue?

**Comment by Maddogofgithub** [comment]
Sorry I accidentally posted without a description
Whenever i start it my hands don't work even if i wait for the game to start
I've tried living large and legacy too.

**Reply by Okabintaro** [reply]
Have you followed the [guide in the README.md](https://github.com/Okabintaro/SubmersedVR#installationsetup) properly?
If you are on quest with link/airlink you also need to add `-vrmode openvr` to the games launch parameters.

**Reply by HyloTheDutchie** [reply, marked as answer]
just found a personal fix, hopefully it works for you-
it turns out my issue was not with SubmersedVR, but it was with BepInEx.  Make sure to go back through and reinstall BepInEx through vortex, and then re-copy the SubmersedVR contents to the subnautica folder.  If it works as it should, when launching Subnautica in VR, you'll see a console open along with the game and you should have a traditional VR select cursor in the main menu.
